version.yaml

```diff
@@ -0,0 +1 @@
+version: v0.5.17
```

Comment:
```suggestion
0.5.17
```

Reply:
can i leave the v in front of it? or is this a hard nogo?

Reply:
nevermind, not as important, that is something i can assume too

Reply:
I'm not sure. You could try it, and we could verify it in the release PR.